requirements.in

```diff
@@ -1,2 +1,3 @@
 pyhcl
 requests
+six
```

Comment:
Do we need to add this to the `install_requires` list within setup.py as well for maximum safety?

Reply:
Absolutely.  I'll add that shortly.